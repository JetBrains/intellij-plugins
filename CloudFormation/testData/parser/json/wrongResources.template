{
  "AWSTemplateFormatVersion": "2010-09-09",
  "Resources": ["XXX"]
}
