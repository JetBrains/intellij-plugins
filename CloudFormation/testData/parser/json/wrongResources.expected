{
  "AWSTemplateFormatVersion": "2010-09-09",
  "Resources": 0@<["XXX"]>
}

0: Expected JSON object
